context/app/static/js/components/Search/styleForTable.js

```diff
@@ -78,6 +79,11 @@ const StyledTableCell = styled(TableCell)`
   }
 `;
 
+const StyledTable = styled(Table)`
+  background-color: white;
+  box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.14), 0 2px 1px -1px rgba(0, 0, 0, 0.12), 0 1px 3px 0 rgba(0, 0, 0, 0.2);
```

Comment:
Do we need to worry about keeping this in sync with the rule for `.sk-layout__filters`? If there's not an easy, better solution, maybe just comments pointing both ways?

Reply:
Actually I can get this value from the theme since it is one of MUI's shadow options. I'll leave a comment inside the `Search.scss` file since it can't access the theme. Sound ok?